ui/js/actions/app.js

```diff
@@ -63,33 +66,31 @@ export function doDownloadUpgrade() {
     const state = getState();
     // Make a new directory within temp directory so the filename is guaranteed to be available
     const dir = fs.mkdtempSync(
-      remote.app.getPath("temp") + require("path").sep
-    ),
+        remote.app.getPath("temp") + require("path").sep
+      ),
       upgradeFilename = selectUpgradeFilename(state);
 
     let options = {
       onProgress: p => dispatch(doUpdateDownloadProgress(Math.round(p * 100))),
       directory: dir,
     };
-    download(
-      remote.getCurrentWindow(),
-      selectUpdateUrl(state),
-      options
-    ).then(downloadItem => {
-      /**
+    download(remote.getCurrentWindow(), selectUpdateUrl(state), options).then(
```

Comment:
Did you make these layout changes or was this Prettier? If it wasn't Prettier, please don't change code formatting and leave it to Prettier.

(Prettier: https://github.com/prettier/prettier)

Reply:
I haven't touched this from what I remember. On my side, there's a TODO there. It may be Prettier.

Reply:
Ok, I'm guessing something just changed such that Prettier reformatted it. You're good here.